utils/zerovec/src/ule/plain.rs

```diff
@@ -98,7 +98,7 @@ macro_rules! impl_const_constructors {
             /// See [`ZeroSlice::cast()`] for an example.
             pub const fn try_from_bytes(bytes: &[u8]) -> Result<&Self, ZeroVecError> {
                 /// core::slice::from_raw_parts(a, b) = core::mem::transmute((a, b)) hack
-                /// ```compile_fail
+                /// ```
```

Comment:
ditto

Reply:
how should it look like here? `core::slice::from_raw_parts` is producing `&[]`, how do you want it casted into `ZeroSlice`?

Reply:
This should call `ZeroSlice::from_bytes_unchecked` after it has done the size check. That seems to be the unchecked version of this.